xcm/xcm-simulator/example/src/lib.rs

```diff
@@ -182,33 +192,201 @@ mod tests {
 			));
 		});
 
-		ParaB::execute_with(|| {
+		ParaB::execute_and_dispatch_xcm(|| {
 			use parachain::{Event, System};
 			assert!(System::events()
 				.iter()
 				.any(|r| matches!(r.event, Event::System(frame_system::Event::Remarked(_, _)))));
 		});
 	}
 
+	/// Scenario:
+	/// A user Alice sends funds from the relaychain to a parachain.
+	///
+	/// Asserts that the correct XCM is sent and the balances are set as expected.
 	#[test]
-	fn reserve_transfer() {
+	fn reserve_transfer_assets() {
```

Comment:
reserve_asset_transfer is correct, since `ReserveAsset` is basically one word, has a specific meaning and should not be separated.

Reply:
The function tested from `pallet_xcm` is named `reserve_transfer_assets` https://github.com/paritytech/polkadot/blob/2882304560ef5b8f2811ddd6b055b78f5de44448/xcm/pallet-xcm/src/lib.rs#L195 

Should this test still be renamed?